frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import { PersonCircleIcon } from '@studio/icons';
+
+type ImageDetails = {
+  src: string;
+  alt: string;
+  title: string;
+};
+
+export type StudioAvatarProps = {
+  imageDetails?: ImageDetails;
+};
+
+export const StudioAvatar = ({ imageDetails }: StudioAvatarProps): ReactElement => {
```

Comment:
Should we add `forwardedRef` to this component as well? We can also use the new `test-utils` to simplify testing the `forwaredRef`. 🥳

Reply:
As discussed on Slack, this made it more complex than necessary 😄